taskfiles/k8s.yml

```diff
@@ -28,6 +30,15 @@ tasks:
           rbac:roleName=manager-role \
           output:crd:artifacts:config=config/crd/bases \
           output:rbac:artifacts:config=config/rbac/bases/operator
+      - |
+        {{.K8S_APPLYCONFIGURATION_GEN_INSTALL_DIR}}/bin/applyconfiguration-gen \
+        --go-header-file "../../../licenses/boilerplate.go.txt" \
+        --output-dir "api/applyconfiguration" \
+        --output-pkg "github.com/redpanda-data/redpanda-operator/src/go/k8s/api/applyconfiguration" \
+        ./api/redpanda/v1alpha2
+      - find ./api/applyconfiguration/redpanda/v1alpha2 -type f -exec sed -i'' 's/"redpanda\/v1/"cluster.redpanda.com\/v1/g' {} \;
```

Comment:
might need to adjust this `sed`/move it into a hack script/go program, but for some reason `applyconfiguration-gen` is setting the `APIVersion` field in the generated code to `redpanda/v1alpha2` rather than `cluster.redpanda.com/v1alpha2`. I don't see a way of overriding it with any sort of command line flags and I'm thinking it might be some weird issue with annotations, but this just manually updates the generated files for the time being.